subsys/bluetooth/host/adv.c

```diff
@@ -1949,13 +1949,13 @@ void bt_hci_le_adv_set_terminated(struct net_buf *buf)
 
 	(void)bt_le_lim_adv_cancel_timeout(adv);
 
-#if (CONFIG_BT_ID_MAX > 1) && (CONFIG_BT_EXT_ADV_MAX_ADV_SET > 1)
+#if (CONFIG_BT_MAX_CONN > 0) && (CONFIG_BT_EXT_ADV_MAX_ADV_SET > 1)
```

Comment:
Same here, and in three other places (if I counted correctly), which I won't separately point out.